hedera-node/configuration/dev/bootstrap.properties

```diff
@@ -1 +1,4 @@
 bootstrap.throttleDefsJson.resource=throttles-dev.json
+contracts.sidecars=CONTRACT_STATE_CHANGE,CONTRACT_BYTECODE
```

Comment:
I'm not sure if we want to update dev, but we do want to update `preprod` which is our integration environment. Someone else could confirm if we also want dev.

Reply:
@stoqnkpL It will be better to update all `bootstrap.properties` in this path `hedera-node/configuration` which will be used by devops. And if we want the CI to use the same properties, we need to update `hedera-services/hedera-node/src/main/resources/bootstrap.properties`